### DISCUSSION TITLE
Rules and scope for user

### DISCUSSION THREAD

**Discussion by ghost** [filebrowser/filebrowser#1740] [Q&A]
I try to configure rules and scope correctly.
I have an admin user who has access to everything
I have normal users who have access to some folders that are not subfolders of their home.

How can I configure this.
I tried with :
scope .
[]ftp.*

for example but it doesn't work

Filebrowser is in a docker.

Thanks for your help

**Comment by CyberGaut** [comment]
### This is still a limit of the system.
You can only specify 1 location so a user _either_ gets their **Home** folder, or gets some **shared** folder
So you could do a setup like this:
     **Admin** .   (everything)
     **User 1** /users/_User1_ (the user 1 home folder)
     **User 2** /_Tank_     (A shared folder)
     **User 3** /_Tank_     (A shared folder)
you can do this to have a lower privileged user
     **Guest** /Tank/_Guest_  (A shared folder)

The **Guest** user can only see the _Guest_ folder, while **User 1** and **User 2** can see everything in _Tank_, including _Guest_
**User 1** can only see their home drive and not _Tank_
**Admin** of course can see everything.
